paddle/fluid/inference/tensorrt/convert/fc_op.cc

```diff
@@ -117,5 +116,5 @@ class FcOpConverter : public OpConverter {
 }  // namespace inference
 }  // namespace paddle
 
-REGISTER_TRT_OP_CONVERTER(fc, FcOpConverter);
+REGISTER_TRT_OP_CONVERTER(mul, FcOpConverter);
```

Comment:
why change `fc` to `mul`?

The fc op is simulated as a `mul(x, w) + zeros`, and `fc` will be mapped by `sub-graph-pass` specificly.

Reply:
Fc OP in fluid is divided into mul op and elementwise add op.  So, there is no fc op in fluid.

Reply:
And ,why we not just use the mul op of tensorrt? because the mul op of the tensorrt jsut support the inputs which have equal rank. (https://docs.nvidia.com/deeplearning/sdk/tensorrt-api/c_api/classnvinfer1_1_1_i_matrix_multiply_layer.html)

Reply:
FC is quite different with MUL, one has parameter, and another has no. 

Leave FC in Fluid(mul with parameters) a FC in TRT, and no need to remove or replace the existing `mul_op`.

Replacing MUL with FC is a trick, and analysis should cover that issue, not in the underlying implementation.